File: scripts/tutorial2/python/plotAvgServiceTime.py
from plotGenericLine import plot_generic_line

if __name__ == '__main__':
    print("--- Generating: Average Service Time Plots ---")

    plot_generic_line(1, 5, 'Service Time (sec)', 'ALL_APPS', '', 'upper left')
    plot_generic_line(2, 5, 'Service Time on Edge (sec)', 'ALL_APPS', '', 'upper left')
    plot_generic_line(3, 5, 'Service Time on Cloud (sec)', 'ALL_APPS', '', 'upper left')

File: scripts/sample_app5/matlab/plotAvgServiceTime.m
function [] = plotAvgServiceTime()

     plotGenericLine(1, 5, 'Average Service Time (sec)', 'ALL_APPS', 'NorthWest', 0);
  
     plotGenericLine(2, 5, 'Service Time on RSU (sec)', 'ALL_APPS', 'NorthWest', 0);

     plotGenericLine(3, 5, 'Service Time on Cloud (sec)', 'ALL_APPS', 'NorthWest', 0);

end